specification/cognitiveservices/data-plane/Personalizer/stable/v1.0/examples/Evaluations_Get.json

```diff
@@ -0,0 +1,69 @@
+{
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "evaluationId": "id",
+    "Ocp-Apim-Subscription-Key": "{API key}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "b58c6d92-b727-48c1-9487-4be2782c9e0a",
+        "name": "myFirstEvaluation",
+        "startTime": "2018-12-19T00:00:00Z",
+        "endTime": "2019-01-19T00:00:00Z",
+        "status": "completed",
+        "policyResults": [
+          {
+            "name": "Custom Policy 1",
```

Comment:
"name": "Custom Learning settings 1",

Reply:
fixed

Reply:
fixed

---
In reply to: [339193691](https://github.com/Azure/azure-rest-api-specs/pull/7624#discussion_r339193691) [](ancestors = 339193691)